src/spec/Parameter.php

```diff
@@ -115,7 +115,20 @@ protected function performValidation()
             }
         }
         if (!empty($this->content) && !empty($this->schema)) {
-            $this->addError("A Parameter Object MUST contain either a schema property, or a content property, but not both. ");
+            $this->addError('A Parameter Object MUST contain either a schema property, or a content property, but not both.');
+        }
+        if (! empty($this->content) && count($this->content) !== 1) {
+            $this->addError('A Parameter Object with Content property MUST have A SINGLE content type.');
+        }
+
+        $supportedSerializationStyles = [
+            'path' => ['simple', 'label', 'matrix'],
+            'query' => ['form', 'spaceDelimited', 'pipeDelimited', 'deepObject'],
+            'header' => ['simple'],
+            'cookie' => ['form'],
+        ];
+        if (! in_array($this->style, $supportedSerializationStyles[$this->in])) {
```

Comment:
`$supportedSerializationStyles[$this->in]` may not be set. An isset() check should be made before this.